### ISSUE TITLE
Can't download paid apps

### ISSUE THREAD

**Issue by barghest** [yeriomin/YalpStore#282]
### Expected behaviour
Download and install already paid apps.

### Actual behaviour
A popup appears that there was a server error:

Yalp version is 0.25

Here's the logcat:
```
build.board: MSM8974
build.bootloader: unknown
build.brand: lge
build.cpu_abi: armeabi-v7a
build.cpu_abi2: armeabi
build.device: d855
build.display: NJH47F
build.fingerprint: lge/g3_global_com/g3:6.0/MRA58K/15351124649f4:user/release-keys
build.hardware: g3
build.host: traian-pc
build.id: NJH47F
build.manufacturer: LGE
build.model: LG-D855
build.product: g3_global_com
build.radio: unknown
build.serial: LGD8552ccf3ba8
build.tags: release-keys
build.time: 1504171408000
build.type: user
build.user: traian
version.codename: REL
version.incremental: eng.traian.20170831.122328
version.release: 7.1.2
version.sdk_int: 25

09-19 07:18:52.949 V/com.github.yeriomin.yalpstore.DetailsActivity(22449): Resuming activity
09-19 07:18:54.051 I/com.github.yeriomin.yalpstore.NativeHttpClientAdapter(22449): Requesting https://android.clients.google.com/fdfe/purchase
09-19 07:18:54.340 I/com.github.yeriomin.yalpstore.NativeHttpClientAdapter(22449): HTTP result code 500
09-19 07:18:54.341 E/com.github.yeriomin.yalpstore.NativeHttpClientAdapter(22449): Exception java.io.FileNotFoundException https://android.clients.google.com/fdfe/purchase
09-19 07:18:54.368 D/com.github.yeriomin.yalpstore.fragment.details.ButtonDownload$LocalPurchaseTask(22449): com.github.yeriomin.playstoreapi.GooglePlayException caught during a google api request: Server error
09-19 07:18:55.782 V/com.github.yeriomin.yalpstore.DetailsActivity(22449): Pausing activity
09-19 07:18:55.783 V/com.github.yeriomin.yalpstore.DetailsActivity(22449): Stopping activity

```

**Comment by nagayev** [comment]
@barghest it was fixed in 0.26.
You can close your issue.

**Comment by barghest** [comment]
Oh, great!
Can't wait for it to land on f-droid.
